pkg/controllers/controllers.go

```diff
@@ -101,12 +101,14 @@ func Register(ctx context.Context, systemNamespace, controllerName string, cfg c
 	klog.Infof("Using default image '%s' for jobs managing helm charts", chart.DefaultJobImage)
 
 	if len(systemNamespace) == 0 {
-		klog.Info("Starting helm controller with no namespace")
+		// TODO: When we add support for setting lock namespace, update "kube-system" to that variable
+		klog.Infof("Starting %s for all namespaces with lock in %s", controllerName, "kube-system")
```

Comment:
import `metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"`
```suggestion
		systemNamespace = metav1.NamespaceSystem
		klog.Infof("Starting %s for all namespaces with lock in %s", controllerName, systemNamespace)
```